stable/wordpress/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
I'm not 100% convinced that the chart should be responsible for creating the TLS secret, IMO this is better done out-of-band.

Reply:
Perhaps that's something the user can decide?  There's nothing forcing you to set the secret in this chart.  We're already providing things like a database without giving the user a choice if they want their own database chart, why is giving them a choice on this bad?

Reply:
The main difference I see with a database is that it is specific to the application and required for it. Ingress rules are also specific to the application (there's a route mapping that may change from application to application). However, the TLS certificate secret is specific to a configuration and not the application, and this is why I believe it belongs out-of-band.

Is this a configuration you actually need for your deployments? I just want to make sure that if we decide to add this, there is a usecase for it and it's not just there because it might be useful.

Reply:
@prydonius how should someone do it out of band? Is there a simple case and a more complex case?

It would be nice to make TLS simple. Right now doing TLS with a Wordpress chart is hard and requires a fair amount of k8s knowledge. Even people who know k8s are tripping over it. How can we make it easy? This was one attempt.

Reply:
Its something I've used personally.  Need is a strong word, but it fits nicely for me.

Reply:
@mattfarina It's fairly easy to create a TLS secret using `kubectl` https://github.com/kubernetes/contrib/tree/master/ingress/controllers/nginx/examples/tls. IMO it's easier to do this than to create a YAML file to pass in my cert and key to the chart. To surface this to people who don't know about `kubectl` we can add a step with an example command in the documentation of the chart.

Nevertheless if others feel that this makes it easier to provide certs, then I'm game.